src/command/feed/upload.ts

```diff
@@ -0,0 +1,54 @@
+import { stat } from 'fs/promises'
+import { LeafCommand, Option } from 'furious-commander'
+import { join } from 'path'
+import { fileExists } from '../../utils'
+import { Upload as FileUpload } from '../upload'
+import { FeedCommand } from './feed-command'
+
+export class Upload extends FeedCommand implements LeafCommand {
+  public readonly name = 'upload'
+
+  public readonly description = 'Upload to a feed'
+
+  @Option({ key: 'path', describe: 'Path of the file', required: true })
+  public path!: string
+
+  @Option({
+    key: 'index-document',
+    describe: 'Default retrieval file on bzz request without provided filepath',
+  })
+  public indexDocument!: string | undefined
+
+  public async run(): Promise<void> {
+    super.init()
+    await this.checkIdentity()
+    const reference = await this.runUpload()
+    await this.updateFeedAndPrint(reference)
+    this.console.dim('Successfully uploaded to feed.')
+  }
+
+  private async runUpload(): Promise<string> {
+    const upload = new FileUpload()
+    const stats = await stat(this.path)
+
+    if (stats.isFile()) {
+      upload.uploadAsFileList = true
+      upload.indexDocument = this.path
```

Comment:
the path of the file is not surely a relative link originated from its parent folder it also could be a full path as well, thereby I don't think it can be good.

Reply:
This was handled with a `basename` on the other side, but for clarity I moved it here now.